scripts/check_test_inclusion.py

```diff
@@ -0,0 +1,89 @@
+#!/bin/python
+
+"""
+Verifies that all tests are a part of the project file.
+"""
+
+from __future__ import print_function
+import os
+import os.path
+import re
+import sys
+
+
+# Tests that are known not to compile in Xcode and can't be added there.
+EXCLUDED = frozenset([
+    # b/79496027
+    'Firestore/core/test/firebase/firestore/remote/serializer_test.cc',
+])
+
+
+def Main(args):
+    problems = CheckProject('Firestore/Example/Firestore.xcodeproj/project.pbxproj',
+                            'Firestore/Example/Tests', 'Firestore/core/test')
+
+    problems = FilterProblems(problems)
+    problems.sort()
```

Comment:
`CheckProject` already returns a sorted list (and `FilterProblems` doesn't alter the order) so I think this is a noop. You might consider dropping one of the sorts.

Reply:
Good call. Refactored a bit more to make this cleaner.